camera_tilt_angle: 60 # degree 60
camera_vertical_FOV: 50 # degree
camera_horizontal_FOV: 59 # degree
camera_view_distance: 25 # meter
camera_offset_x: 0 # meter
camera_offset_y: 0 # meter
camera_offset_z: 0 # meter 

duplicate_margin_object: 5 # meter
duplicate_margin_goal: 0.5 # meter
evaluate_pair_threshold: 25 # meter
goal_extend_length: 0.01 # meter
red_totem: [
  [65.38, 47.37, 0],
  [74.17, 58.7, 0],
  [63.63, 67.95, 0],
  [52, 82.13, 0],
  [46.3, 101.2, 0],
  [51.8, 123.2, 0],
]

green_totem: [
  [70.31, 34.41, 0],
  [86.46, 53.88, 0],
  [76.85, 73.88, 0],
  [69.18, 88.5, 0],
  [58.7, 98.3, 0],
  [65.7, 111.6, 0],
]

white_totem: [
  [50, 140, 0],
  [65, 135, 0],
  [50.5, 52.15, 0],
  [42.64, 37.09, 0],
]